src/Public/OpenApi.ps1

```diff
@@ -1343,6 +1343,10 @@ function Set-PodeOARouteInfo
         [string]
         $Description,
 
+        [Parameter()]
+        [string]
+        $OperationId,
```

Comment:
This will need a new `.PARAMETER` comment above the function,

Reply:
I added a short description.